std/format/read.d

```diff
@@ -1,17 +1,49 @@
 // Written in the D programming language.
 
 /**
-   This is a submodule of $(MREF std, format).
-   It provides some helpful tools.
+This is a submodule of $(MREF std, format).
 
-   Copyright: Copyright The D Language Foundation 2000-2013.
+It provides two functions for reading formatted input: $(LREF
+unformatValue) and $(LREF formattedRead). The former reads a single
+value. The latter reads several values at once and matches the
+characters found between format specifiers.
 
-   License: $(HTTP boost.org/LICENSE_1_0.txt, Boost License 1.0).
+For reading purpose, only part of the possibilities of $(MREF_ALTTEXT
+format strings, std,format) is used: The parameters are almost
```

Comment:
What do you mean by "the parameters are completely ignored" ?

Reply:
It doesn't matter if you write `%-#030.10,7d` or `%d`. There is no difference. The only exception is `%*d`.

Reply:
I see. Maybe this rewording sounds better: "Format specifiers are ignored, except for the ones starting with a single $(B '*'). See $(LREF formattedRead) for more information." ?

Reply:
Seems like we've got some babel confusion here - see the discussion in #7935. Maybe we need to rethink the wording of the grammar?

According to the (new) grammar, the "format specifier" includes the letter at the end, which is not ignored, only the parameters of that specifier. Ideally, we should link each of these words directly to the definition in the grammar, but as far as I can see, there is currently no way to do so correctly in ddoc and ddox at the same time; maybe not at all in ddox.

Suggestion: "Parameters are ignored, except for the ones consisting of a single $(B '*'). See $(LREF formattedRead) for more information."

Reply:
Yeah, I saw the discussion after I left the previous comment. I agree with the suggestion.